rdrf/rdrf/templates/rdrf_cdes/alert_language_is_not_supported.html

```diff
@@ -0,0 +1,26 @@
+{% load get_language_settings_codes %}
+{% get_language_settings_codes as LANGUAGES %}
+
+<script type="text/javascript">
+    $(document).ready(function(){
+        const browser_language = navigator.language;
+        const browser_language_in_settings = [browser_language, browser_language.split('-')[0]].some((language) => {{ LANGUAGES|safe }}.includes(language));
```

Comment:
suggestion: It helps with code readability to prefix boolean variables with `is` or `has` e.g. `is_browser_language_in_settings`.